docs/tutorial.md

```diff
@@ -1,3 +1,55 @@
 # Tutorial
 
-TODO Use demo_plugin to describe how to do stuff.
+To help illustrate the concepts of NewHELM, we provide a series of functional (if silly) examples in the `demo_plugin` folder. This tutorial walks through those examples.
+
+# Tests
+
+## Creating a basic Test
+
+[Demo: DemoSimpleQATest](../demo_plugin/newhelm/tests/demo_01_simple_qa_test.py)
+
+Lets say we want to create a Test where we send a bunch of questions to the SUT, and expect it to respond with specific answers. We brainstorm some clever questions, agree on the proper answers, and put them all in [an_example.jsonl](https://storage.googleapis.com/mlc-aisafety-scratch-dev-7292542/an_example.jsonl).
+
+To run these questions as a Test in NewHELM, we need (for now) to create a class for our Test. Lets call our Test `DemoSimpleQATest`. 
+Since this fits as a [Prompt Response Tests](prompt_response_tests.md) we can have it inherit from `BasePromptResponseTest`. We now have several abstract methods we need to define.
+
+The first phase in a PromptResponseTest is making the `TestItem`s. We want these to be our questions from `an_example.jsonl`. NewHELM uses [DependencyHelper](../newhelm/dependency_helper.py) to ensure good hygiene of data dependencies (e.g. versioning). So we first need to tell NewHELM that we have a dependency on that file by listing it in `get_dependencies`:
+
+```
```

Comment:
This is for all the code snippets, but Github flavored markdown supports highlighting:

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
```
def get_dependencies(self):
    return {
        "jsonl_questions": WebData(
            source_url="https://storage.googleapis.com/mlc-aisafety-scratch-dev-7292542/an_example.jsonl"
        ),
    }
 ```
 
 ```py
 def get_dependencies(self):
    return {
        "jsonl_questions": WebData(
            source_url="https://storage.googleapis.com/mlc-aisafety-scratch-dev-7292542/an_example.jsonl"
        ),
    }
```

Reply:
Neat! Done.